README.md

```
@@ -1,23 +1,115 @@
+JZFS is a data management software designed to support the various stages
+of the development of digital objects.
+
+Importantly, JZFS can be seen as an overlay on top of existing data
+structures and services: Tracking files does not change the files themselves or the location from which they can
+be retrieved by data processing tools.
+
+JZFS is used to collect
+all experimental metadata about the complete timeline of longitudinal and multimodal animal experiments,
+including MRI, histology, electrophysiology, and behavior.
 
+![](./docs/jzfs-research-flow.png)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)